src/new-components/modal/modal-stories-components.tsx

```diff
@@ -17,13 +17,15 @@ function Link({ children, ...props }: JSX.IntrinsicElements['a']) {
 
 type ModalStoryState = Pick<ModalProps, 'width' | 'height'> & {
     button: 'true' | 'false' | 'custom'
+    hideOn: 'escapeAndOverlay' | 'escape' | 'overlay'
```

Comment:
💭 Not super important but I would prefer to see this follow the same naming as the props, ie. `escapeAndInteractOutside` and `interactOutside`